src/amo/components/AddonReview.js

```diff
@@ -104,15 +94,6 @@ export class AddonReviewBase extends React.Component {
         <form onSubmit={this.onSubmit} ref={(ref) => { this.reviewForm = ref; }}>
           <div className="AddonReview-form-input">
             {errorHandler.hasError() ? errorHandler.renderError() : null}
-            <input
```

Comment:
The textarea doesn't have a label in the mock. Are you suggesting we should add a hidden one? I'm not too familiar with a11y -- what purpose does that serve?